mipengine/tests/node/test_views.py

```diff
@@ -3,99 +3,56 @@
 import pymonetdb
 import pytest
 
-from mipengine.node.tasks.data_classes import ColumnInfo
-from mipengine.node.tasks.data_classes import TableData
-from mipengine.node.tasks.data_classes import TableSchema
-from mipengine.tests.node.set_up_nodes import celery_local_node_1
+from mipengine.common.node_tasks_DTOs import ColumnInfo
+from mipengine.common.node_tasks_DTOs import TableData
+from mipengine.common.node_tasks_DTOs import TableSchema
+from mipengine.tests.node import nodes_communication
 
-create_table = celery_local_node_1.signature('mipengine.node.tasks.tables.create_table')
-create_view = celery_local_node_1.signature('mipengine.node.tasks.views.create_view')
-get_views = celery_local_node_1.signature('mipengine.node.tasks.views.get_views')
-get_view_data = celery_local_node_1.signature('mipengine.node.tasks.views.get_view_data')
-get_view_schema = celery_local_node_1.signature('mipengine.node.tasks.views.get_view_schema')
-clean_up = celery_local_node_1.signature('mipengine.node.tasks.common.clean_up')
+local_node_id = "local_node_1"
+local_node = nodes_communication.get_celery_app(local_node_id)
+local_node_create_view = nodes_communication.get_celery_create_view_signature(local_node)
+local_node_get_views = nodes_communication.get_celery_get_views_signature(local_node)
+local_node_get_view_data = nodes_communication.get_celery_get_table_data_signature(local_node)
+local_node_get_view_schema = nodes_communication.get_celery_get_table_schema_signature(local_node)
+local_node_cleanup = nodes_communication.get_celery_cleanup_signature(local_node)
 
+context_id = "regrEssion"
 
-def test_views():
-    context_id = "regrEssion"
+
+@pytest.fixture(autouse=True)
+def cleanup_views():
+    yield
+
+    local_node_cleanup.delay(context_id=context_id.lower()).get()
```

Comment:
You defined exactly the same function [here](https://github.com/madgik/MIP-Engine/blob/eb063dc8eacb6d91df61f12cb789bb31b9745496/mipengine/tests/node/test_merge_tables.py#L21)

Reply:
They both use different global env variables so it's not that simple, we need to pass parameters. Will get back to it if something more than a cleanup is needed.